server/webhook.go

```diff
@@ -199,3 +207,44 @@ func (p *Plugin) GetWebhookURL(jiraURL string, teamID, channelID string) (subURL
 
 	return subURL, legacyURL, nil
 }
+
+func (wh *webhook) applyReporterNotification(bb []byte, reporter *jira.User) error {
+	if wh.eventTypes.ContainsAny("event_created_comment", "event_updated_comment") {
+		jwhook := wh.JiraWebhook
+		err := json.Unmarshal(bb, &jwhook)
+		if err != nil {
+			return err
+		}
+		if jwhook.Issue.ID == "" {
+			return ErrWebhookIgnored
```

Comment:
We won't ever reach this point since the initial webhook parser returns this error already in this case